english/bite-the-marshmallow/tasty-marshmallow.md

```diff
@@ -1,3 +1,5 @@
+[A nose length away from the wall, a twinkle in your mind makes you pause.](padded-room/first-blackout)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.